pub/spec/fixtures/pubspec_yamlfiles/git_https.yaml

```diff
@@ -0,0 +1,15 @@
+name: git_https
+description: A simple command-line application.
+version: 1.0.0
+homepage: https://www.example.com
+
+environment:
+  sdk: '>=2.10.0 <3.0.0'
+
+dependencies:
+  path:
+    git: https://github.com/dart-lang/path.git
+
+dev_dependencies:
+  pedantic:
+    git: https://github.com/google/pedantic.git
```

Comment:
Instead of using git paths, why not use what's found on pub.dev?

This applies to all instances where this occurs not just in this file.

Reply:
Those are just for testing.

Some do use dependencies from git and these pubspec files are exactly to cover those use-cases. 👍